assets/charts/control-plane/kubelet/templates/kubelet-ds.yaml

```diff
@@ -65,36 +65,50 @@ spec:
         securityContext:
           privileged: true
         volumeMounts:
+        # Used by Calico.
         - mountPath: /var/lib/cni
           name: coreos-var-lib-cni
           readOnly: false
+        # This is in needed so that the calico CNI plugin can read the file named `nodename` in the
+        # following path. This file is created by `calico-node` daemonset pod.
         - mountPath: /var/lib/calico
           name: coreos-var-lib-calico
           readOnly: true
+        # Used by Calico.
         - mountPath: /opt/cni/bin
           name: coreos-opt-cni-bin
           readOnly: true
-        # TODO check if this is needed
         - name: dev
           mountPath: /dev
+          readOnly: false
+        # Here kubelet stores the lock file and unix sockets.
         - name: run
           mountPath: /run
+          readOnly: false
         - name: sys
           mountPath: /sys
+          readOnly: false
+        # This is mounted so that node local storage works fine.
         - name: mnt
           mountPath: /mnt
           mountPropagation: Bidirectional
+        # This directory has certs that kubelet needs to authenticate.
         - name: etc-kubernetes
           mountPath: /etc/kubernetes
           readOnly: true
+        # This is needed so that kubelet can access the logs(symlinked) of the containers. The
+        # CAdvisor(which is baked into kubelet) needs access to monitor the container and report via
```

Comment:
```suggestion
        # CAdvisor (which is baked into kubelet) needs access to monitor the container and report via
```

Reply:
Why should there be a space before bracket?

Reply:
- https://oilpatchwriting.wordpress.com/2010/09/13/spaces-around-parentheses/#:~:text=Here's%20the%20rule%3A,and%20no%20space%20before%20it.&text=Note%20that%20the%20period%20at,sentence%20is%20outside%20the%20parentheses.
- https://english.stackexchange.com/questions/5987/is-there-any-rule-for-the-placement-of-space-after-and-before-parentheses
- https://www.quora.com/Is-not-putting-a-space-immediately-before-an-open-parenthesis-becoming-more-popular